logtracing/trace.go

```diff
@@ -167,26 +186,26 @@ func LogSpan(ctx context.Context, s *span) {
 
 	keyvals = append(keyvals, s.keyvals...)
 
-	if s.err != nil {
+	if s.panic != nil {
 		keyvals = append(keyvals,
-			"msg", fmt.Sprintf("%s (%v) -> %s (%T)", s.name, dur, s.err, s.err),
-			"span.err", s.err,
-			"span.err_type", errType(s.err),
+			"msg", fmt.Sprintf("%s (%v) -> panic: %s (%T)", s.name, dur, s.panic, s.panic),
```

Comment:
use `%v` or `%+v` instead of `%s` for `s.panic`.

Reply:
Replaced with `%+v`